modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/helper/CPContentHelperImpl.java

```diff
@@ -705,6 +680,32 @@ public boolean hasReplacement(
 		return false;
 	}
 
+	@Override
+	public boolean isDirectReplacement(CPSku cpSku) throws Exception {
+		if ((cpSku == null) || cpSku.isDiscontinued()) {
```

Comment:
a discontinued sku may be a replacement for another sku

Reply:
```suggestion
		if (cpSku == null) {
```

Reply:
@andrea-ale-sbarra , the method is only being used in the JSP that handles whether the replacementTab should show.
Do we still want to show the replacement tab and all of the replacements even if the SKU is discontinued?

Reply:
Yes, why not? In the API I believe we are returning these information, so I believe we should do the same for the UI. Anyway this is more a question for the product, do you mind ask the same to David?
Thank you!